external/PATCH-01-STRATUM

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021 Intel Corporation.
+# Copyright (c) 2021, 2022 Intel Corporation.
```

Comment:
1. This is generally written Copyright (c) <first year>-<last year>. The comma form is more common when there are gaps in the timeline.
2. I would recommend putting the SPDX license ID after the copyright notice and before the license text. SPDX IDs are for ease of use. They should be easy to find. Putting something after the license text makes it harder to find.

Reply:
Addressed the concerns. Fixed.